sdkit/utils/latent_utils.py

```diff
@@ -1,59 +1,87 @@
+"""
+    Latent utils
+"""
 import numpy as np
 import torch
 from PIL import Image, ImageOps
 from einops import repeat, rearrange
 
 from sdkit import Context
 
-def img_to_tensor(img: Image, batch_size, device, half_precision: bool, shift_range=False, unsqueeze=False):
+
+def img_to_tensor(
+    img: Image.Image,
+    batch_size,
+    device,
+    half_precision: bool,
+    shift_range=False,
+    unsqueeze=False
+):
+    """ Convert PIL image to torch tensor. """
     if img is None:
         return None
 
-    img = np.array(img).astype(np.float32) / 255.0
-    img = img[None].transpose(0, 3, 1, 2)
-    img = torch.from_numpy(img)
-    img = 2. * img - 1. if shift_range else img
-    img = img.to(device)
+    _img: np.ndarray = np.array(img).astype(np.float32) / 255.0
```

Comment:
Why are we using `_img` here?

Reply:
The local var was reused. Not sure why this was chosen.